src/main/java/com/academia/library/repository/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.academia.library.repository;
+
+import com.academia.library.model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
Why did you use this annotation here only ?

Reply:
Ok, thank